### ISSUE TITLE
Benchmarking on Graviton nodes

### ISSUE THREAD

**Issue by Cai-Chen** [opensearch-project/OpenSearch#2337]
I am comparing benchmark results on Graviton VS x86.

- Tool: rally
- Track: pmc
-  Instance: r5.2xlarge(volume IOPS 4500 in gp2) VS r6g.2xlarge(volume IOPS 4500 in gp3)

**Result**

Mean Throughput
Operation | type | unit | r5.2xlarge | r6g.2xlarge
-- | -- | -- | -- | --
index-append | Batch | docs/s | 1982.46 | 2865.43
default | Interactive | ops/s | 19.98 | 19.96
term | Interactive | ops/s | 19.99 | 19.96
phrase | Interactive | ops/s | 19.99 | 19.97
articles_monthly_agg_uncached | Interactive | ops/s | 19.93 | 19.94
articles_monthly_agg_cached | Interactive | ops/s | 20 | 19.99
scroll | Interactive | pages/s | 12.56 | 12.56

90th percentile latency 
Operation | type | unit | r5.2xlarge | r6g.2xlarge
-- | -- | -- | -- | --
default | Interactive | ms | 14.16 | 16.71
term | Interactive | ms | 13.56 | 15.08
phrase | Interactive | ms | 13.39 | 13.89
articles_monthly_agg_uncached | Interactive | ms | 16.10 | 16.44
articles_monthly_agg_cached | Interactive | ms | 7.54 | 8.25
scroll | Interactive | ms | 782.27 | 877.14
index-append | Batch | ms | 3,666.45 | 2,545.25

index-append is better for throughput and latency on Graviton. But x86 performs better for other operations in terms of latency.

Has anyone got any ideas? Thanks

**Comment by jkowall** [comment]
Can you pass along the rally parameters you used? I looked at that track: https://github.com/elastic/rally-tracks/tree/master/pmc you only have 5 shards on this, and likely that wouldn't take advantage of the fact that Graviton has more threads per core. Additionally, you are using EBS and I would highly suggest using local SSD for these types of workloads.

**Comment by Cai-Chen** [comment]
> Can you pass along the rally parameters you used? I looked at that track: https://github.com/elastic/rally-tracks/tree/master/pmc you only have 5 shards on this, and likely that wouldn't take advantage of the fact that Graviton has more threads per core. Additionally, you are using EBS and I would highly suggest using local SSD for these types of workloads.

Yes, I am using pmc with default challenge `append-no-conflicts`. Nothing specific in rally parameters, its like 
`docker run elastic/rally:2.1.0 race --challenge=append-no-conflicts  --track=http_logs --target-hosts=host1, host2, host3--pipeline=benchmark-only --client options="use_ssl:true,verify_certs :false,basic_auth_user:'xxxx', basic_auth_password:'xxxxx'"`

What is the recommended number of shards on Graviton? And can you give more detail in which size of workload would be better to use EBS?

**Comment by anasalkouz** [comment]
I think we should document a recommended cluster setup per node type.
